task_02/tevfik_aksoy/main.cpp

```diff
@@ -0,0 +1,183 @@
+#include <iostream>
+#include <vector>
+#include <string>
+#include <tuple>
+#include <set>
+#include <fstream>
+
+using std::cout;
+using std::endl;
+using std::get;
+using std::to_string;
+using std::vector;
+
+//Tuple of <int,int> for realization of vector of vectors of <int,int>
+typedef std::tuple<int, int> DestID_EdgeID;
+
+//Structure of nodes(vertices)
+struct Node
+{
+private:
+	int id = 0;

```

Comment:
`int` было бы хорошо заменить на что-нибудь более выразительное, например:
```cpp
using NodeId = int;
```
В таком случае и читаемость кода будет лучше, и если мы захотим сделать `id` например строкой, достаточно будет изменить только в одном месте. 
```suggestion
      NodeId id = 0;
```